package.json

```diff
@@ -5,12 +5,11 @@
   "private": true,
   "scripts": {
     "postinstall": "cd confidentiality-contracts && yarn && yarn build",
-    "onboard": "ts-node src/main.ts",
     "erc20": "ts-node src/main.ts erc20",
     "dataOnChain": "ts-node src/main.ts dataOnChain"
   },
   "devDependencies": {
-    "@coti-io/coti-sdk-typescript": "https://github.com/coti-io/coti-sdk-typescript.git",
+    "@coti-io/coti-sdk-typescript": "https://github.com/coti-io/coti-sdk-typescript.git#11a38cf",
```

Comment:
why is this pointing to a specific commit?

Reply:
This example uses the SDK and it relied on pending changes from my PR in the TypeScript SDK repo to work properly. I mentioned this to you on Slack.